src/net/sf/memoranda/ui/AgendaPanel.java

```diff
@@ -48,7 +48,7 @@
 	JButton historyForwardB = new JButton();
 	JButton export = new JButton();
 	JEditorPane viewer = new JEditorPane("text/html", "");
-	String[] priorities = {"Muy Alta","Alta","Media","Baja","Muy Baja"};
+/*TJ*/	String[] priorities = {"Very High","High","Medium","Low","Very Low"};

```

Comment:
Language changes reviewed by ghgofort 12:23am - 2/8/16. All changes look ready to commit.